### PR TITLE
NextJS Starbucks App

### PR DESCRIPTION
**Pull request by sebko**
Trello URL: https://trello.com/c/nac1VAKh/4512-port-starbucks-app-to-nextjs

Card to update examples section in dev docs: https://trello.com/c/SqCSlC1e/4513-dev-docs-site-add-starbucks-app-to-menu-examples-section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,77 @@
...
+_Example menuId: 3404a063-2bbe-4473-bb1b-cac3c7e1d14e_
+
+**Localhost**
+
+- Main Menu Board: `http://localhost:3000?espressoAndCoffeeSubheading={espressoAndCoffeeSubheading}&espressoAndCoffeeBottomText={espressoAndCoffeeBottomText}&footnote={footnote}&menu={menuId}`
+
+**Production**
+
+<!-- - Main Menu Board: https://custom-menu-app-1.netlify.app?espressoAndCoffeeSubheading=<espressoAndCoffeeSubheading>&espressoAndCoffeeBottomText=<espressoAndCoffeeBottomText>&
+  footnote=<footnote>&menuId={menuId} -->
```
Comment (sebko) [inline README.md]: I tried to setup the production deploy with Vercel, but can't seem to add this repo to Vercel. 

Screen Shot 2022-07-28 at 2 48 22 pm

After clicking "Update access" the modal disappears but the repo does not appear on the list.

File: README.md [hunk 2]
```diff
@@ -1 +1,77 @@
...
+
+## Development
+
+1. Run `cp .env.sample .env.local` and populate env vars
+
+2. Run the development server
+
+```bash
+yarn dev
+```
```
Comment (jschr) [inline README.md]: Nice 💯 

I was a bit lazy with the READMEs for the other [sample apps](https://github.com/mirainc/simple-weather-app). Mind making a Trello card (or two) to improve the READMEs of the other sample apps with what you've done here?

File: pages/index.js [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+  const router = useRouter();
+
+  useEffect(() => {
+    const refreshData = () => {
+      router.replace(router.asPath);
+    };
+
+    const id = setInterval(() => {
+      refreshData();
+    }, 30000);
```
Comment (jschr) [inline pages/index.js]: Just confirming, was this 30s in the original starbucks app? Looks like it's 30s in the whole foods app but wonder if we should set it to 10s to make demos a little nicer. No preference on my end really, up to you!

**Review by jschr — APPROVED** [review]
Nice!

File: README.md (hunk 2 above)
Comment (sebko) [reply README.md]: Np! I've updated the README for the Wholefoods app already. Thanks again for porting those over 🙏 

https://trello.com/c/B1rRV71P/4532-sample-apps-update-readme-for-simple-weather-app

File: pages/index.js (hunk 3 above)
Comment (sebko) [reply pages/index.js]: My mistake! I'll update this to 10 seconds. Thanks for picking that up 👍

**Pull request merged**